trusted_os/main.go

```diff
@@ -42,9 +45,15 @@ var (
 )
 
 var (
-	Network = usbarmory.ENET2
-	Storage = usbarmory.MMC
-	Control = usbarmory.USB1
+	Control *usb.USB
+	Storage *usdhc.USDHC
+
+	// USB armory Mk II (rev. γ) - UA-MKII-γ
+	// USB armory Mk II (rev. γ) - UA-MKII-γ
```

Comment:
Why the duplicate comment? I assumed that this would be referencing the USB support for the other board, but looking below we don't init this field for the LAN version.

Reply:
Typo [fixed](https://github.com/transparency-dev/armored-witness-os/pull/40/commits/1cdb0dced88f6f164c6a11d657b03c5d9a52ee0f).